screenpy_selenium/actions/save_console_log.py

```diff
@@ -80,15 +84,16 @@ def and_attach_it(self, **kwargs: Any) -> Self:  # noqa: ANN401
     def perform_as(self, the_actor: Actor) -> None:
         """Direct the actor to save their browser's console log."""
         browser = the_actor.ability_to(BrowseTheWeb).browser
-        js_log = "\n".join([str(entry) for entry in browser.get_log("browser")])
+        if not isinstance(browser, ChromiumDriver):
+            msg = "Only Chromium-based drivers can save their console logs."
+            raise UnableToAct(msg)
 
-        with open(self.path, "w+", encoding="utf-8") as js_log_file:
-            js_log_file.write(js_log)
+        js_log = "\n".join([str(entry) for entry in browser.get_log("browser")])
+        self.path.write_text(js_log, encoding="utf-8")
 
         if self.attach_kwargs is not None:
-            the_actor.attempts_to(AttachTheFile(self.path, **self.attach_kwargs))
+            the_actor.attempts_to(AttachTheFile(str(self.path), **self.attach_kwargs))
```

Comment:
So, i want to update this to a `Path` in AttachTheFile, but i need to release `screenpy` first. Debating releasing ScreenPy, then updating this bit (and the bit in `SaveScreenshot` too), then releasing this. Gonna have to remember to do that so let's release 5.0 soon!

Reply:
If you want this to be `Path`, then I suggest removing `str()` here and let mypy complain until we release the screenpy update.  In it's current state, the tests are failing because it's expecting `Path` in called args but getting a string.